Bottom: 3ab948dfa83c4e207f070f468c3bbc170fb460b2
Top:    89a11c07e3528284e7e17a2863787d1d3d5f5fb5
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 23:20:57 -0500

Refresh of readme

---

diff --git a/README.md b/README.md
index 99e20e6..0520977 100644
--- a/README.md
+++ b/README.md
@@ -4,4 +4,5 @@ tablecalc
 Dokuwiki plugin which adds limited spreadsheet functionality to tables. See
 the original documentation page at: https://www.dokuwiki.org/plugin:tablecalc
 
-Forked from https://github.com/pirati-cz/tablecalc
+Forked from https://github.com/pirati-cz/tablecalc. See the CHANGELOG.md for a
+list of differences.
